source/linux/background-managers/gnome.js

```diff
@@ -18,7 +30,7 @@ export async function set(imagePath) {
 	await execFile('gsettings', [
 		'set',
 		'org.gnome.desktop.background',
-		'picture-uri',
+		(await isDarkTheme()) ? 'picture-uri-dark' : 'picture-uri',
```

Comment:
Move it to a variable, for consistency with `get`.

Reply:
Do it yourself